panel.py

```diff
@@ -245,6 +245,8 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		if fdt.getprop_or_none(node, 'qcom,mdss-dsi-tx-eot-append') is None:
 			self.flags.append('MIPI_DSI_MODE_NO_EOT_PACKET')
 
+		self.cphy_mode = fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None
```

Comment:
Can you move this line above `# Check DSI controller if LDO mode is needed`? Seems a bit weird stuffed in the middle of the flags. (Sorry for not noticing this earlier)

Reply:
Ok, I will rebase it.